src/include/ipebitmap.h

```diff
@@ -69,6 +70,8 @@ namespace ipe {
 
     inline bool isJpeg() const;
     inline bool isGray() const;
+    inline bool isExternal() const;
+    inline String getExternalPath() const;
```

Comment:
Ipe doesn't use the `get` prefix for getters - this should simply be `externalPath`, or even just `path`.